script.js

```diff
@@ -142,4 +142,20 @@ form.addEventListener('submit', (e) => {
   } else {
     form.submit();
   }
-});
\ No newline at end of file
+});
+
+// local storage
+
```

Comment:
- [ ] Good job so far :clap: But according to the project requirement you need to create **a single JavaScript object with all the data from the entire form** and save it in local storage. Please don't create one local storage entry per input field. Please fix it in order to make your contact form works as expected :+1: 

```
const formArray = {
    name: value,
    email: value,
    message: value,
};
```